source/pcic/client.py

```diff
@@ -6,17 +6,19 @@
 import json
 import re
 import io
-
+import o2x5xx
```

Comment:
import from source instead of an installed library is better i think

Reply:
@Kottanidisan see comment

Reply:
Just to confirm, the `import` statements need to be left as they are.

For testing changes like these I suggest to create a virtual environment where you install the o2x5xx-python packages and try a simple `import o2x5xx` afterwards. With the current state of the branch to be merged you will get the error
```
    from o2x5xx import json, re, io
ImportError: cannot import name 'json' from partially initialized module 'o2x5xx' (most likely due to a circular import) (/tmp/venv/lib/python3.10/site-packages/o2x5xx-0.2-py3.10.egg/o2x5xx/__init__.py)
```

Reply:
@Kottanidisan Maybe a check like this can be done automatically by github for a branch?